src/bitcoind.ts

```diff
@@ -87,10 +92,22 @@ export class BitcoindClient {
     hexstring,
   }: {
     hexstring: string
-  }): Promise<{ vout: [VOut] }> {
+  }): Promise<DecodeRawTransactionResult> {
     return await this.client.decodeRawTransaction({ hexstring })
   }
 
+  // default client knows about all transactions of its wallets
+
+  async getTransaction({
+    txid,
+    include_watchonly,
+  }: {
+    txid: string
+    include_watchonly?: boolean
```

Comment:
This shouldn't be nullable. Should always be either `true` or `false`. If you don't always want to pass it best would be to set a default like `include_watchonly = true`

Reply:
Done